frontend/src/pages/Login.tsx

```diff
@@ -0,0 +1,91 @@
+import { useNavigate } from 'react-router-dom';
+import { getAuth, signInWithEmailAndPassword } from "firebase/auth";
+import React from 'react';
+import PageTemplate from '../components/PageTemplate';
+
+function Login () {
+  const [email, setEmail] = React.useState('');
```

Comment:
Can you just `import { useState } from 'react'` so you don't have to preface everything with React.